get_png_files_from_drive.py

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env python3
+
+# Copyright 2020 Google, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+from __future__ import print_function
+import pickle
+from os import path, makedirs, walk, listdir
+import io
+import fire
+import sys
+import shutil
+from distutils.dir_util import copy_tree
+from googleapiclient.discovery import build
+from google_auth_oauthlib.flow import InstalledAppFlow
+from google.auth.transport.requests import Request
+from googleapiclient.http import MediaIoBaseDownload
+
+# If modifying these scopes, delete the file token.pickle.
+SCOPES = ["https://www.googleapis.com/auth/drive"]
+
+
+def main(folder_name="", reporting=False):
+
+    # Remove combined_png dir if it exists
+    if path.exists("./build/combined_png"):
+        shutil.rmtree("./build/combined_png")
+
+    # Create a token.pickle file to store the users session
+    service = get_service()
+
+    if folder_name == "":
+        folder_name = input("Please enter the name of your drive folder: ")
```

Comment:
No user input blocks in things that run in make. Just error out and tell the user very clearly how to establish a drive folder.

Reply:
We can remove it entirely. We already show an error somewhere else:
`sys.exit(f'{total} folders found, needs exactly one')`